migration/fmtp/CsvStringifyModified.js

```diff
@@ -223,7 +223,7 @@
      if (columns) {
        for (i = j = 0, ref = columns.length; 0 <= ref ? j < ref : j > ref; i = 0 <= ref ? ++j : --j) {
          column = columns[i];
-         _line[i] = typeof line[column] === 'undefined' || line[column] === null ? '' : line[column];
+         _line[i] = typeof line[column] === 'undefined' ? null : line[column];
```

Comment:
Right now, all empty values are converted to empty strings. We need to preserve NULLs so they are different from empty values.